_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -61,7 +65,9 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 1. Launch Mario Kart 7
   - Ensure that wireless connectivity is enabled
   - kartdlphax should tell you that it is running
-1. Navigate to `Local Multiplayer` -> `Create Group`
+1. Select `Local Multiplayer`, then select the type of the target console, either 
+1. and `3DS ROP xPloit Injector`, then `Use Settings`
```

Comment:
what happened here

Reply:
i didnt know what to write there, seems like i forgot to change it back, should be fixed now?